integration/spring-boot/starter-jaxws/pom.xml

```diff
@@ -54,6 +55,10 @@
                 </exclusion>
             </exclusions>
         </dependency>
+        <dependency>
```

Comment:
This should be optional dependency, correct? (not everyone would need actuator)

Reply:
Yes it should. I've removed that and moved it into integration test's pom.xml. It uses that.